layouts/partials/tw/donate-prefill-lg.html

```diff
@@ -0,0 +1,209 @@
+{{ $wrapperClass := .wrapperClass  | default ""}}
+{{ $donateHed := .donateHed | default "Join our historic effort to invigorate local news." }}
+{{ $donateCta := .donateCta | default "Contribute" }}
+{{ $donateGaCategory := .donateGaCategory | default "donate-prefill-lg" }}
+{{ $donateGaForm := .donateGaForm | default "" }}
+{{ $donateCampaign := .donateCampaign | default "7015G0000013oHVQAY" }}
+{{ $donateTheme := .donateTheme | default "" }}
+{{ $donateAmounts := .donateAmounts | default (slice "10" "15" "25" "50") }}
+
+
+{{ $btnClass := ` mr-4 flex-[0_1_30%] outline-none min-w-[180px] inline-block rounded py-3 px-6 text-white font-sans font-black text-lg tracking-wide ring ring-transparent` }}
+{{ $activeClass := `bg-red-6 text-white hover:bg-red-5 focus:ring-red-7 active:ring-red-9` }}
+{{ $inactiveClass := `bg-s-2 text-s-9 hover:bg-s-3 focus:ring-s-7 active:ring-s-9` }}
+{{ $radioClass  :=  `rounded-full border-s-1 appearance-none mr-4 order-first w-6 h-6 checked:bg-black shadow-[0_0_0_3px_rgba(255,255,255,1)]` }}
+
+{{ $donateAmount := split $donateAmounts "," }}
+{{ $donateFirst := index $donateAmounts 1 }}
+{{ $donateDefault := int $donateFirst }}
+<div
+  id="donate-leader"
```

Comment:
Do we need this id? Probably we'll only have one of these on the page, but I don't like that it limits us if we wanted more.